RMSSD Function.R

```diff
@@ -0,0 +1,185 @@
+#RMSSD Function
+rm(list=ls())
+
+dataset <- `2019.7.10_RR_p36t0`
+
+#HR data may be outputed from equipment as RR values or ms/time values
+#let the individual specify what type of data they have
+
+data <- as.numeric(dataset[,2])
+x <- data.frame(data)
+
+
+
+
+#RMSSD FUNCTION
+#most common measure of heart rate variability 
+myRMSSD <- function(x, input, segment, max, min) {  
```

Comment:
based on the R style code (http://adv-r.had.co.nz/Style.html), function names should be verbs and objects nouns. Maybe you could name your function getRMSSD() or calculateRMSSD()?